docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1301,11 +1273,11 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Reason:**
 
-  TODO
+  Graph with broken dependencies of the plugins.
```

Comment:
Its interesting that ZWED0152I isnt a warning or error.
Actually you can remove it, its a DEBUG.